Header header

float32[] x_components
float32[] y_components

float32[] x_comp_eigenvector
float32[] y_comp_eigenvector
float32[] eigenvalues

int32[] usefulLaserBeans #those greater than range_min

